fdbserver/DeltaTree.h

```diff
@@ -1699,7 +1699,7 @@ struct DeltaTree2 {
 		int count = end - begin;
 		numItems = count;
 		nodeBytesDeleted = 0;
-		initialHeight = (uint8_t)log2(count) + 1;
+		initialHeight = count == 0 ? 0 : (uint8_t)log2(count) + 1;
```

Comment:
I'm not sure, but it would not matter for correctness.  There are no critical decisions based on this height, it is only used to compare the max height at a later time to the initial height as a heuristic for whether or not the tree is too unbalanced and should be rebuilt to balance it.  The worst outcome of initial height being wrong would be rebuilding the page more or less often than the correct number would.

That said, it would be better to use something like `ctz(count + 1) + 1`, but I'm not going to worry about it for this PR.  This change was just to fix an undefined behavior which `ubsan` found.